packages/terra-date-picker/src/react-datepicker/month.jsx

```diff
@@ -156,6 +156,12 @@ export default class Month extends React.Component {
     }
   }
 
+  handleMonthFocus = () => {
+    if (this.props.onMonthFocus) {
+      this.props.onMonthFocus()
```

Comment:
We should add `onMonthFocus` as a first class prop.

Reply:
[Added...!](https://github.com/cerner/terra-framework/pull/1134/commits/fc3dc12a0ef35935fc6e47719fa2a110acd313e1#diff-df786ca21c3be186c57955b173f95877)